homeassistant/components/aladdin_connect/cover.py

```diff
@@ -66,10 +66,10 @@ async def async_setup_entry(
     try:
         doors = await hass.async_add_executor_job(acc.get_doors)
 
-    except (TypeError, KeyError, NameError, ValueError) as ex:
-        _LOGGER.error("%s", ex)
-        raise ConfigEntryNotReady from ex
-
+    except ValueError as ex:
```

Comment:
It doesn't look like ValueError is possible here either.

https://github.com/shoejosh/aladdin-connect/blob/a351c6ce6c407e255a4954cd05b3c2fdb367f99d/aladdin_connect/__init__.py#L59-L68

https://github.com/shoejosh/aladdin-connect/blob/a351c6ce6c407e255a4954cd05b3c2fdb367f99d/aladdin_connect/__init__.py#L70-L93

Reply:
Sure enough... wonder why this test ever lived in the original code?!?!